README.md

```diff
@@ -69,7 +69,8 @@ ______________________________________________________________________
 ### create a query engine in seconds
 
 ```python
->>> from autollm import AutoQueryEngine, read_files_as_documents
+>>> from autollm.utils.document_reading import read_files_as_documents
+>>> from autollm import AutoQueryEngine
```

Comment:
```suggestion
>>> from autollm import AutoQueryEngine, read_files_as_documents
```
this line still works @AbrahimZaman360. We have made it simpler for quickstarters to import our frequently used classes and functions via adding  to our [init](https://github.com/safevideo/autollm/blob/main/autollm/__init__.py#L11-L21) file. Thank you anyway for your interest in autollm!

If you have further improvements or feedback, please feel free to share!

Reply:
didn't work on my side! that's why suggested (:

Reply:
Hi @AbrahimZaman360,

I am curious about the issue you ran into since the import should work as is. To get to the bottom of this, could you share:

- The version of python and `autollm` you're using:
```bash
python --version
```
```python
import autollm
print(autollm.__version__)
```
- Detailed error message when trying the original import.

This info will help us understand what's happening. Your efforts to improve the project are much appreciated!